packages/sit-onyx/package.json

```diff
@@ -34,6 +34,7 @@
     "url": "https://github.com/SchwarzIT/onyx/issues"
   },
   "scripts": {
+    "td": "turbo watch --filter=sit-onyx dev",
```

Comment:
What is this script and when should it be used?

Reply:
This script watches all dependencies of `sit-onyx#dev` and runs them